### DISCUSSION TITLE
Running the OG-PHL model

### DISCUSSION THREAD

**Discussion by rickecon** [EAPD-DRB/OG-PHL#20] [General]
I ran the model successfully last night.
- I deleted the `ogphl-dev` conda environment: `conda remove --name ogphl-dev --all`
- I re-created a fresh version of the `ogphl-dev` conda environment: `conda env create -f environment.yml` (this took about 10 minutes)
- I ran the `run_og_phl.py` example script.
    - the baseline steady-state equilibrium took less than a minute to solve
    - the baseline transition path equilibrium took 23 iterations and 33 minutes to solve
    - the reform steady-state equilibrium took less than a minute to solve
    - the reform transition path equilibrium to 23 iterations and 33 minutes to solve

**Comment by marcelolafleur** [comment]
@rickecon I tried to do this from a fresh install. 

- Fully updated conda and anaconda
- sync repo and cloned OG-PHL
- I re-created a fresh version of the ogphl-dev conda environment: `conda env create -f environment.yml`. For me this only took 1 minute but it said it completed. 
- `pip install -e .`
- Got similar times as you

**Comment by jmsf1005** [comment]
Hi @rickecon I also ran the model using run_og_phl.py script 

- I got the similar times as you on the baseline steady-state equilibrium and reform steady-state equilibrium
- The baseline transition path equilibrium and the reform transition path equilibrium took 23 iterations and 24 minutes to solve
